oracle/price-feeder/README.md

```diff
@@ -121,3 +121,31 @@ Ex :
 `export PRICE_FEEDER_PASS=keyringPassword`
 
 If this environment variable is not set, the price feeder will prompt the user for input.
+
+## Systemd Configuration
+
+In order to run the price feeder as a background process, you can set up a systemd service for it. Here is an example of the service that will run the price feeder process:
+
+```ini
+[Unit]
+Description=Oracle Price Feeder
+After=network.target
+
+[Service]
+User=root
+Type=simple
+Environment="PRICE_FEEDER_PASS={KEYRING_PASSWORD}"
+ExecStart=/root/go/bin/price-feeder {PATH-TO-CONFIG-TOML}
```

Comment:
how do we get the config-toml?

Reply:
i think we should also add instructions for building/running the price-feeder? I'm looking at the sei-infra instructions and it seems more involved than just setting up the systemd process 

`cd /home/ubuntu/sei-chain/; make install-price-feeder`